### ISSUE TITLE
Zoomify issue[bug] 

### ISSUE THREAD

**Issue by F1reRa1n** [Siphalor/mouse-wheelie#258]
MC 1.20.4
Mouse Wheelie 1.13.3
Zoomify 2.13.0

With both Mouse Wheelie and Zoomify active, holding "C" results in insane levels of zoom. Disabling Zoomify entirely results in zoom still occurring when pressing "C". Disabling Mouse Wheelie results in Zoomify working as intended with rational levels of zoom. There are no settings in the Mouse Wheelie config relating to zoom.

**Comment by Siphalor** [comment]
Hi, I'm honestly not sure what's going on there.
Mouse Wheelie has no code affecting zooming at all.

Zoomify actually supports different levels of zoom, have you tried scrolling while pressing C?

**Comment by F1reRa1n** [comment]
Yeah, I did try that. I also tried resetting Zoomify to default settings which should be only x4 zoom. I also tried disabling scroll zoom entirely as well. The only solution I found to rectify the issue is to disable Mouse Wheelie.
